src/main/java/dan200/computercraft/shared/common/BlockGeneric.java

```diff
@@ -219,12 +219,6 @@ public final AxisAlignedBB getBoundingBox( IBlockState state, IBlockAccess world
     }
 
     @Override
-    public final AxisAlignedBB getSelectedBoundingBox( IBlockState state, World world, BlockPos pos )
```

Comment:
Rather than deleting this method, could we implement it by calling getBoundingBox() and adding the blockpos?